src/util/getBlobURL.ts

```diff
@@ -0,0 +1,14 @@
+import { RuleFile } from './constants';
+import { env } from '../environment';
+import { logger } from './debug';
+
+const debug = logger('getBlobURL');
+
+export const getBlobURL = (filename: string, files: RuleFile[], owner: string, repo: string, base: string): string => {
```

Comment:
Can you add a docstring here explaining what this does?

Reply:
good call. just to clarify, I'm extracting this from statuses so we can use it anywhere in the codebase. 
I thought it will be nice to click on the rule (filename or description) and go to the rule itself. 

did you see that?